src/components/ui/BreadCrumbs.tsx

```diff
@@ -22,27 +22,36 @@ interface BreakCrumbsProps {
 
 function BreadCrumbs ({ pathTokens, ancestors, isClimbPage = false }: BreakCrumbsProps): JSX.Element {
   return (
-    <div aria-label='area-breadcrumbs'>
+    <div aria-label='area-breadcrumbs' className='flex-wrap flex gap-2 text-sm items-center'>
+      <svg xmlns='http://www.w3.org/2000/svg' className='h-5 w-5 text-ob-primary' fill='none' viewBox='0 0 24 24' stroke='currentColor' strokeWidth={2}>
```

Comment:
Is this the location marker?  Would it be better to use `<LocationIcon />` from the @heroicon icon package (already part of the project dep) to improved readability/semantic?

Reply:
Good catch, I'll fix that